config/log4j.properties

```diff
@@ -55,6 +55,20 @@ log4j.appender.authorizerAppender.File=${kafka.logs.dir}/kafka-authorizer.log
 log4j.appender.authorizerAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.authorizerAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
 
+log4j.appender.metricsAppender=org.apache.log4j.DailyRollingFileAppender
+log4j.appender.metricsAppender.DatePattern='.'yyyy-MM-dd-HH
+log4j.appender.metricsAppender.File=${kafka.logs.dir}/BrokerMetrics.log
+log4j.appender.metricsAppender.layout=org.apache.log4j.PatternLayout
+log4j.appender.metricsAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+// define KAFKA appender to a KAFKA topic
+log4j.appender.KAFKA=org.apache.kafka.log4jappender.KafkaLog4jAppender
+log4j.appender.KAFKA.BrokerList=localhost:9092
+log4j.appender.KAFKA.Topic=BrokerMetrics
```

Comment:
Do we think this is a safe choice? I think we need to be pretty careful naming this topic in order to do our best to avoid any potential conflicts.

Kafka internal topics and other projects have just prefixed with `_` or `__` to try to avoid conflicts.